var/spack/repos/builtin/packages/libfabric/package.py

```diff
@@ -156,12 +156,12 @@ def setup_build_environment(self, env):
     # To enable this package add it to the LD_LIBRARY_PATH
     def setup_run_environment(self, env):
         libfabric_home = self.spec["libfabric"].prefix
-        env.prepend_path("LD_LIBRARY_PATH", libfabric_home.lib)
+        env.prepend_path("LD_LIBRARY_PATH", libfabric_home.lib64 + ":" + libfabric_home.lib)
 
     # To enable this package add it to the LD_LIBRARY_PATH
     def setup_dependent_run_environment(self, env, dependent_spec):
         libfabric_home = self.spec["libfabric"].prefix
-        env.prepend_path("LD_LIBRARY_PATH", libfabric_home.lib)
+        env.prepend_path("LD_LIBRARY_PATH", libfabric_home.lib64 + ":" + libfabric_home.lib)
```

Comment:
do two prepends (we don't know the OS path separator)

Reply:
Done.